queries/gotmpl/highlights.scm

```diff
@@ -0,0 +1,117 @@
+; Identifiers
+
+([
+  (field)
+  (field_identifier)
+  ] @variable.member (#set! priority 105))
```

Comment:
I guess we can now move the priorities to the nivm-treesitter repo and delete the queries here.

Reply:
I opened a [PR](https://github.com/nvim-treesitter/nvim-treesitter/pull/6712) for the queries in nvim-treesitter.